shared/src/persistence/dynamo/documents/deleteDocketEntry.ts

```diff
@@ -17,7 +17,7 @@ export const deleteDocketEntry = async ({
   docketEntryId: string;
   docketNumber: string;
 }) =>
-  remove({
+  await remove({
```

Comment:
might more more concise to remove the `async` instead of add the `await`. Both will effectively return a promise.